packages/frontend/src/layout/CollectionPage/CollectionPage.jsx

```diff
@@ -26,7 +26,9 @@ const EMPTY_COLLECTION = {
 function getShareableURLForUnauthenticated(collectionName, datasetIds) {
   const urlOrigin = window.location.origin;
   const datasetIdsStr = datasetIds.join(',');
-  return `${urlOrigin}/collection/${collectionName}/${datasetIdsStr}`;
+  return `${urlOrigin}/collection/${encodeURIComponent(
```

Comment:
Can we extract the `encodeURIComponent` call to a separate variable (e.g. 'const encodedCollectedName') just so that we don't end up with these line breaks that makes the interpolated string harder to read?